src/components/tips/tip-card.tsx

```diff
@@ -29,24 +27,15 @@ const TipCard: React.FC<{tip: Tip; i: number}> = ({tip, i}) => {
   const tipTopic = tip?.tags ? tip.tags[0]?.name : ''
 
   return (
-    <Link
+    <article
       className={twMerge(
         cx(
-          'group relative flex flex-row items-center overflow-hidden rounded border border-transparent bg-white hover:bg-gray-100 dark:bg-gray-400/5 hover:dark:bg-gray-800 transition duration-300 ease-in-out',
+          'group relative flex flex-row items-center overflow-hidden rounded border border-transparent bg-white dark:bg-gray-400/5 transition duration-300 ease-in-out px-4 py-2 min-h-[160px] ',
```

Comment:
min height here is nice because ~160px is around the height of the average tip with title being one row and description being two